docs/user/openstack/README.md

```diff
@@ -68,14 +68,22 @@ openstack role add --user <user> --project <project> swiftoperator
 openstack object store account set --property Temp-URL-Key=superkey
 ```
 
-**NOTE:** Swift is required as the user-data provided by OpenStack is not big enough to store the ignition config files, so they are served by Swift instead.
+**NOTE:** Swift is required as the user-data provided by OpenStack Metadata service is not big enough to store the ignition config files, so they are served by Swift instead.
 
-* You may need to increase the security group related quotas from their default   values. For example (as an OpenStack administrator):
+You may need to increase the security group related quotas from their default values. For example (as an OpenStack administrator):
```

Comment:
This seems to belong to a different (sub)section, or within the quotas section, instead of inside Swift requirements

Reply:
Indeed, I'll move it where it belongs.